### PR TITLE
Adding SDC metric and tested

### LINKED ISSUE #73: metric_Sørensen–Dice_coefficient
**Update metadata:**
https://docs.google.com/spreadsheets/d/1QCeAF4yQG4bhZSGPQwwVBj_XF7ADY_2mK5xivAIfHsc/edit#gid=4776337

**Details of metric (publication, GitHub):**
closely related to Jaccard

### PR DESCRIPTION
**Pull request by Jieran-S**
Implemented sørensendice_coefficient(SDC) metric to close #73 

Due to the similarity of this metric to Jaccard score, the evaluation function is adopted from [`sklearn.metrics.jaccard_score`](http://scikit-learn.org/stable/modules/generated/sklearn.metrics.jaccard_score.html). It has also already been tested.

### CODE REVIEW

**Comment by niklasmueboe** [comment]
Please remove all `.ipynb_checkpoint` files before a review will be started

**Comment by Jieran-S** [comment]
Hi, all `ipynb_checkpoints` file should be deleted! :))

**Review by niklasmueboe — COMMENTED** [review]
Isn't this the same as the F1 score?

**Comment by Jieran-S** [comment]
Yea it's the same! If we have F1 score maybe we can close it? I thought we didn't implement F1 score....

**Comment by niklasmueboe** [comment]
We have a domain/cluster-specific F1 score (which also includes the overall score I think) already. So I guess this can be closed
